supervisor/shared/display.c

```diff
@@ -45,8 +45,10 @@
 #include "shared-module/sharpdisplay/SharpMemoryFramebuffer.h"
 #endif
 
+#ifndef CIRCUITPY_REMOVE_REPL_LOGO
```

Comment:
Please switch these to reading the value and then putting the default in `py/circuitpy_mpconfig.h`. I also inverted the name since positive flags are easier to reason about.

```suggestion
#if CIRCUITPY_REPL_LOGO
```